tests/conftest.py

```diff
@@ -37,6 +37,7 @@
 
 setup_logging("logs/unit_tests.log")
 
+from src.mechaclient import MechaClient
```

Comment:
[Style] (unused-import) What does adding this import provide? I can't see any additional code that uses this import.

Reply:
Id argue this is convention / refactor more than it is style, but yes, this appears to be a needless import.

Reply:
If its an invalid import, I'll remove it.  I'll double check.

Reply:
unused import removed in 96dac7b.

Reply:
I believe you removed the wrong import; the one I was referring to is in `conftest.py`, which is still there.

Reply:
This resolved?